cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -14,7 +15,12 @@
 
 {% block content_main %}
     {% for block in page.header -%}
-        {% if loop.index == 1 %}
+        {% if 'featured_content' in block.block_type %}
```

Comment:
The condition is to run the macro, this is necessary for modules that we need to work in both the wagtail and non-wagtail pages.